examples/next/pages/ui/components/authenticator/sign-in-totp-mfa/index.page.tsx

```diff
@@ -1,17 +1,24 @@
 import { Amplify } from 'aws-amplify';
 
-import { Authenticator } from '@aws-amplify/ui-react';
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react';
 import '@aws-amplify/ui-react/styles.css';
 
 import awsExports from './aws-exports';
 Amplify.configure(awsExports);
 
 export default function SignInTotpMfa() {
   const formFields = { setupTOTP: { QR: { totpIssuer: 'My Web App' } } };
+  const { user } = useAuthenticator((context) => [context.user]);
+
   return (
     <>
       <Authenticator formFields={formFields}>
-        {({ signOut }) => <button onClick={signOut}>Sign out</button>}
+        {({ signOut }) => (
```

Comment:
I passed the `useAuthenticator` hook because it's closer to what the customer saw when using `useAuthenticator`. On the other hand the `user` being passed in with the children function should be the same. I'm ok switching it to passing user here.

Reply:
Fixed in 491a2eb887221b699a3745ed82861f3bf1122320